### ISSUE TITLE
Text 'Responsible Computing and User Agreement' is not announced as heading via voiceover on the 'Terms of use' page.

### ISSUE THREAD

**Issue by msftedad** [gdh1995/pdf.js#19]
Attach (recommended) or Link to PDF file here:
 https://mozilla.github.io/pdf.js/web/viewer.html.
[edad (2).pdf](https://github.com/gdh1995/pdf.js/files/14600407/edad.2.pdf)

Configuration:
- Operating system and its version: iPhone 13 Version-17.4
- PDF.js version: 1.4   

Pre-requisites:
Open Settings> Accessibility> Voiceover> Turn on Voiceover.

Steps to reproduce the problem:

1. Open this PDF viewer https://mozilla.github.io/pdf.js/web/viewer.html.
2. Navigate to 'Tools' option at header at header of pdf viewer.
3. Activate 'Open' control and upload any pdf from your system.
4. Now you'll have the pdf file, in pdf viewer.
5. Navigate through the headings and observe the issue.

What is the expected behavior? (add screenshot)
Text 'Responsible Computing and User Agreement' should be defined as heading as this is the title of the document.

What went wrong? (add screenshot)
Text 'Responsible Computing and User Agreement' is not announced as heading via voiceover on the 'Terms of use' page.

https://github.com/gdh1995/pdf.js/assets/93735775/764c316e-41a5-479f-b4e0-a6de3f1c877c

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by msftedad** [comment]
Hi @gdh1995, Any update on this issue? Please provide the ETA to fix this issue.
